docs/content/tutorial/default.yml

```diff
@@ -207,7 +207,7 @@ body:
         "name": "Nico Williams"
       }
       {
-        "message": "Fix iterration problem for non decimal string\n\nWhen the string transformation to number failed, all following\ntransformation failed too.\n\nThis happend because status in decNumberFromString function is\nupdated just in error case. Reusing the DEC_CONTEXT that failed\nbefore results into error even if the string is valid number.",
```

Comment:
This is a part of a real commit message, so it should be as it is like `iterration`?

Reply:
Honestly, I think we could create a better tutorial without using GitHub REST API, but that's off-topic of this PR.